### DISCUSSION TITLE
Does Tab support add/close buttons like most browsers?

### DISCUSSION THREAD

**Discussion by HubKing** [Megabit/Blazorise#4361] [Q&A] [answered]
Basically, what I want is a tab that works like modern web browsers. [+] for adding a new tab, and [X] for closing a tab. Is that supported?

![image](https://user-images.githubusercontent.com/10900489/205510239-648bda29-fc6b-485e-bb6a-e2c6a1e6c9ad.png)
![image](https://user-images.githubusercontent.com/10900489/205510265-e6f634a3-d225-4b42-9aca-3d3a5ec0b282.png)

https://blazorise.com/docs/components/tab I could not find examples here.

**Comment by stsrki** [comment, marked as answer]
Out of the box, no. Our tabs are quite _atomic_ in design, meaning they don't have much logic behind them. You can probably add a close button yourself and adjust it with CSS to the right side. And for the closing of tabs, you will also need to do it yourself. Basically, you will not render those that are closed. Or set them `Display="Display.None"`.

**Reply by David-Moreira** [reply]
@stsrki would you want to keep the tabs very simple? Or take this as a feature request? Seems like a nice to have.

I actually happen to have a personal tabs component that mimicks that exact browser behaviour.
https://david-moreira.github.io/BlazorXTabs/

But I also advise to modify the `Blazorise Tabs` in order to obtain the desired behaviour.
